src/components/ChannelSearch/ChannelSearch.tsx

```diff
@@ -23,4 +19,6 @@ const ChannelSearch = () => (
   </div>
 );
 
-export default React.memo(ChannelSearch);
+export const ChannelSearch = React.memo(
```

Comment:
does this do anything if the component doesn't have props?

Reply:
I believe it was being used before so kept it. but it should still cache the component in memory for future renders of parent components. so it doesn't have to be recreated at higher levels